telegram/_bot.py

```diff
@@ -5951,8 +5951,9 @@ async def send_poll(
                 :tg-const:`telegram.Poll.REGULAR`, defaults to :tg-const:`telegram.Poll.REGULAR`.
             allows_multiple_answers (:obj:`bool`, optional): :obj:`True`, if the poll allows
                 multiple answers, ignored for polls in quiz mode, defaults to :obj:`False`.
-            correct_option_id (:obj:`int`, optional): 0-based identifier of the correct answer
-                option, required for polls in quiz mode.
+            correct_option_id (:obj:`int`): Optional. A zero based identifier of the correct answer
+                option. Available only for closed polls in the quiz mode, which were sent (not 
+                forwarded), by the bot or to a private chat with the bot.
```

Comment:
Bot methods should stick to official documentation from core.telegram.org/bots/api

Please revert

Reply:
done